Common/src/grid_movement_structure.cpp

```diff
@@ -9160,11 +9160,6 @@ void CElasticityMovement::SetVolume_Deformation_Elas(CGeometry *geometry, CConfi
 
   bool discrete_adjoint = config->GetDiscrete_Adjoint();
 
-  int rank = MASTER_NODE;
```

Comment:
Hey Tim,
I've noticed that you remove this statements all over the place. However, "rank" and "rank=MASTER_NODE" is still widely used, how does the code access the variable?